src/components/modals/ConditionModal.tsx

```diff
@@ -227,26 +231,26 @@ const Component: React.FC<Props> = (props) => {
             return
         }
 
-        const condition: CLM.Condition = {
+        const cond: CLM.Condition = {
             entityId: selectedEntityOption.data.entityId,
             condition: selectedOperatorOption.data
         }
 
         if (showNumberValue) {
-            condition.value = numberValue
+            cond.value = numberValue
         }
         else if (selectedEnumValueOption) {
             // TODO: Fix enum types
-            condition.valueId = selectedEnumValueOption.data.enumValueId!
+            cond.valueId = selectedEnumValueOption.data.enumValueId!
         }
 
         return condition
```

Comment:
Return value looks to be incorrect.  Should the "cond" with the var name change

Reply:
Thanks, good catch!